docs/reference/query-dsl/terms-query.asciidoc

```diff
@@ -7,7 +7,8 @@
 Returns documents that contain one or more *exact* terms in a provided field.
 
 The `terms` query is the same as the <<query-dsl-term-query, `term` query>>,
-except you can search for multiple values.
+except you can search for a match in multiple values. If you need more
+than one matching term at once, you may find <<query-dsl-terms-set-query, `terms_set` query>> useful.
```

Comment:
```suggestion
except you can search for multiple values. A document will match if it contains
at least one of the terms. To search for documents that contain more than one
matching term, use the <<query-dsl-terms-set-query, `terms_set` query>>.
```

Reply:
Thanks for this contribution @etki ! I've a left a suggestion that I think makes the difference between the two queries a bit more clear. Let me know what you think.